redaxo/src/core/lib/sql/table.php

```diff
@@ -145,10 +145,10 @@ private function __construct($name)
      *
      * @return self
      */
-    public static function get($name)
+    public static function get($name, $dbid = 1)
     {
-        return self::getInstance($name, static function ($name) {
-            return new self($name);
+        return self::getInstance([$dbid,$name], static function ($dbid, $name) {
```

Comment:
Ist die reihenfolge der parameter vertauscht?

Reply:
Ist das nicht in sich geschlossen: https://github.com/redaxo/redaxo/blob/a3b98ea907a2344836de626ce95c01391b958792/redaxo/src/core/lib/base/instance_pool_trait.php#L62
Evtl. etwas verwirrend, dass das gedreht ist.

Reply:
Wobei es ja mein Vorschlag in Slack war, diese Reihenfolge zu nehmen.
Fand es logischer, erst die DB, dann die Tabelle, im Key zu haben.
Aber letztlich wurscht.

Technisch jedenfalls passt es so, die Parameter hinten müssen in der Reihenfolge der Arrayelemente vorne vom `$key` sein.

Reply:
ich kann mir vorstellen, dass @staabm meinte, dass die Reihenfolge abweichend vom Constructor ist (hat zumindest mich erst verwirrt). Die Parameter werden aber Richtig in Z151 übergeben.